### PR TITLE
Chorus without Faust

### PR DESCRIPTION
**Pull request by brunosilvano**
So this is a work in progress for the Chorus effect using the Gamma lib. It is already in a working condition, but it still needs some improvements for setting the params (when changing the depth or delay really fast, there are some cracks in the sound) and also to set a nice range for the Props.
A feedback would be much appreciated!

### CODE REVIEW

**Comment by jmidt** [comment]
I made a change to the way the depth parameter is updated. I can approve this now.

**Comment by topisani** [comment]
travis says this wont compile - i think theres a closing curly brace too much. Fix that, and we're ready to merge

File: src/engines/fx/chorus/chorus.cpp [hunk 1]
```diff
@@ -5,42 +5,41 @@
...
-    auto& props = engine.props;
-    switch (ev.rotary) {
-    case Rotary::blue: props.delay.step(ev.clicks); break;
-    case Rotary::green: props.rate.step(ev.clicks); break;
-    case Rotary::yellow: props.deviation.step(ev.clicks); break;
-    case Rotary::red: props.depth.step(ev.clicks); break;
+      props.depth.on_change().connect([this](float depth) {
+        chorus.depth(depth);
+      });
     }
```
Comment (topisani) [inline src/engines/fx/chorus/chorus.cpp]: Thats the one - delete this line and it should work

Comment (jmidt) [reply src/engines/fx/chorus/chorus.cpp]: Done. Waiting for travis.

**Comment by brunosilvano** [comment]
Found some other issues, just fixed them now.

**Pull request merged**